### PR TITLE
Add Freeze Frame (sorry for repost)

### PR DESCRIPTION
**Pull request by Armodillomatt12**
### Prerequisites
* [x] I have read and understood the [contribution guidelines](CONTRIBUTING.md)
* [x] This pull request follows the code style of the project
* [x] I have tested this feature thoroughly WITH THE USE of the load-all-sounds starter
  * [Link to a video of proof of testing](https://www.youtube.com/watch?v=7HolFz9raQw)

**Issues Fixed:** [Any issues fixed by this pull request]

### Changes Proposed in this Pull Request
* added freeze frame to `games.json` and added cues for it as well.

...
P.S. i had to make a new fork because the one i had is outdated. sorry if i'm being annoying. also, i deleted the `games.json` from the freeze frame folder i made. i forgot that it's still in that folder. `games.json` in the data folder was not deleted.

### CODE REVIEW

**Comment by GuardedLolz** [comment]
You deleted Animal Acrobat, Tram and Pauline, Airboarder, and Space Soccer.

**Review by GuardedLolz — CHANGES_REQUESTED** [review]
DO NOT remove games.

**Comment by GuardedLolz** [comment]
Now you just have to wait.

**Comment by Armodillomatt12** [comment]
sorry...

**Comment by Armodillomatt12** [comment]
i accidentally put `games.json` to the `freezeFrame` folder. i didn't delete `games.json` from data, @GuardedLolz. i should had been more specific.

**Comment by GuardedLolz** [comment]
Also, I think the yellow cars should be 0.5 beats. That would save us the trouble of using tracks.

**Review by GuardedLolz — CHANGES_REQUESTED** [review]
The yellow cars need to be 0.5 beats.

File: core/assets/sounds/cues/freezeFrame/data.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      "cues": [
+        {
+          "id": "freezeFrame/beginningSignal1",
+        "beat": 0.0,
+	"track": 0
+        },
+        {
+          "id": "freezeFrame/beginningSignal1",
+          "beat": 1.0,
+  "track": 0
```
Comment (chrislo27) [inline core/assets/sounds/cues/freezeFrame/data.json]: Formatting

File: core/assets/sounds/cues/freezeFrame/data.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        },
+        {
+          "id": "freezeFrame/beginningSignal2",
+          "beat": 2.0,
+  "track": 0
```
Comment (chrislo27) [inline core/assets/sounds/cues/freezeFrame/data.json]: Formatting

File: core/assets/sounds/cues/freezeFrame/data.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      "name": "fast car zoom"
+    },
+    {
+      "id": "freezeFrame/fastCarClick",
+      "duration": 2.0,
+      "name": "fast car click"
+    },
+    {
+      "id": "freezeFrame/smallCarZoom",
+      "duration": 1.0,
```
Comment (chrislo27) [inline core/assets/sounds/cues/freezeFrame/data.json]: Should be 0.5 beats

File: core/assets/sounds/cues/freezeFrame/data.json [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+       },
+       {
+        "id": "freezeFrame/smallCarZoom",
+        "beat": 1.0,
+	"track": 0
+       },
+       {
+        "id": "freezeFrame/smallCarZoom",
+        "beat": 0.5,
+	"track": 1
```
Comment (chrislo27) [inline core/assets/sounds/cues/freezeFrame/data.json]: Keep it one-dimensional (assuming you're also making it 0.5 beats as mentioned)

**Review by chrislo27 — CHANGES_REQUESTED** [review]

File: core/assets/sounds/cues/freezeFrame/data.json (hunk 2 above)
Comment (Armodillomatt12) [reply core/assets/sounds/cues/freezeFrame/data.json]: what do you by formatting?

**Comment by Armodillomatt12** [comment]
here the link of the cues you wanted me to change for the small cars. https://www.youtube.com/watch?v=8bIm5Y9rzxE

File: core/assets/sounds/cues/freezeFrame/data.json (hunk 2 above)
Comment (GuardedLolz) [reply core/assets/sounds/cues/freezeFrame/data.json]: He means you should put spaces there.

Comment (Armodillomatt12) [reply core/assets/sounds/cues/freezeFrame/data.json]: beginning signal 1 and 2?

Comment (Armodillomatt12) [reply core/assets/sounds/cues/freezeFrame/data.json]: knowing that we don't use spaces.

**Pull request merged**